src/queue.js

```diff
@@ -0,0 +1,44 @@
+export default class Queue {
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  enqueue(element) {
+    return this.elements[this.size++] = element
```

Comment:
I didn't realize you could increment a variable while using it as a key. Well done 👍